cpp/src/parquet/stream_reader_test.cc

```diff
@@ -947,5 +947,77 @@ TEST_F(TestReadingDataFiles, ByteArrayDecimal) {
   EXPECT_EQ(i, 25);
 }
 
+class TestMultiRowGroupStreamReader : public ::testing::Test {
+ public:
+  TestMultiRowGroupStreamReader() { createTestFile(); }
+
+ protected:
+  const char* GetDataFile() const { return "stream_reader_multirowgroup_test.parquet"; }
```

Comment:
It would be better to write this to an in-memory file.

Reply:
I am happy to make that change but as this is my first contribution I'm not sure how to do this. This test fixture is simply a copy/paste of the existing `TestStreamReader` fixture above but with different initialization code.  Could you provide an example of how to do an in-memory file?

Reply:
I see. That's fine. Thanks!